firebase-firestore-mixin.html

```diff
@@ -169,10 +169,11 @@
 
         this._firestoreProps = {};
         this._firestoreListeners = {};
-        this.db = this.constructor.db || firebase.firestore();
       }
 
       connectedCallback() {
+        super.connectedCallback();
```

Comment:
May I ask why did you decide to move it here? What’s the reasoning behind it?

Reply:
I mainly stole these two lines from #279, but the goal is to not have an issue like this: https://receptive-kitten.glitch.me/ I can look into what this is actually fixing soon.

Reply:
Is it actually needed to fix it? I can’t really see why would it help.

Reply:
Ok, finally got into this part more. It helps because if you allow `super.connectedCallback()` to run, then the code at https://github.com/firebase/polymerfire/blob/master/firebase-app.html#L123 will also be allowed to fully run, which means that the app will have actually connected to Firebase. It's possible that  you suggestions on moving some of the `this.db` stuff out to the scope might address this as well. However, were that not so, maybe this is a use case for https://www.polymer-project.org/2.0/docs/api/namespaces/Polymer.RenderStatus#function-Polymer.RenderStatus.afterNextRender ?